src/chainparams.cpp

```diff
@@ -166,10 +166,10 @@ class CMainParams : public CChainParams {
         };
 
         chainTxData = ChainTxData{
-            // Data from RPC: getchaintxstats 4096 00000000000000000008a89e854d57e5667df88f1cdef6fde2fbca1de5b639ad
-            /* nTime    */ 1626697539,
-            /* nTxCount */ 656509474,
-            /* dTxRate  */ 2.424920418708139,
+            // Data from RPC: getchaintxstats 4320 000000000000000000052d314a259755ca65944e68df6b12a067ea8f1f5a7091
+            /* nTime    */ 1645542140,
+            /* nTxCount */ 712531200,
+            /* dTxRate  */ 2.911573625926285,
```

Comment:
d8a8b54 update `ChainTxData` for testnet and signet too? (they were last updated in eeddd1c8fa9)

same for `nMinimumChainWork` and `defaultAssumeValid`

Reply:
(side note, I have 41.4 GiB of `/indexes`, should that space be taken into account anywhere)

Reply:
Could do it seperately. I don't have any testnet node at the moment,

> (side note, I have 41.4 GiB of /indexes on mainnet (4 GiB on testnet), wonder if that space ought to be taken into account somewhere)

Good point, but I'm simply following the release process here. It could be updated to include indexes, but I don't have any indexes myself, I guess we assume a default setup without them? That makes some sense for new users (remember, this is for the GUI).